### PR TITLE
Adding a tutorials and starters page 

### PR DESCRIPTION
**Pull request by mzkrasner**
# [starters and tutorials page] - #[mzk/starters-tutorials]

## Description

A simple additional page was added to capture starters and tutorials (made both by Ceramic and our community) to serve readers with a commonly-used set of boilerplate code and configurations.

### CODE REVIEW

File: website/versioned_docs/version-0.5.x/examples.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- [**Build an AI Chatbot on ComposeDB**](https://learnweb3.io/lessons/build-an-ai-chatbot-on-compose-db-and-the-ceramic-network) - Read a LearnWeb3.io tutorial on how the `OpenAI Realtime Chat with ComposeDB` repository (linked above) works and how to set it up.
+- [**Query Filtering and Ordering in ComposeDB**](https://blog.ceramic.network/tutorial-query-filtering-and-ordering-in-composedb/) - Learn the ins and outs of filtering and ordering based on schema subfields in this tutorial blog post.
+- [**Creating `MetIRL` Attestations with EAS**](https://docs.attest.sh/docs/tutorials/ceramic-storage) - Learn how the `Ethereum Attestation Service on ComposeDB` repository (linked above) works, how to generate attestations, and how to create confirmations.
+- [**Encrypted Data on ComposeDB**](https://blog.ceramic.network/tutorial-encrypted-data-on-composedb/) - Learn yet another way to encrypt and decrypt data on ComposeDB by generating an `encryptionDid` instance.
+
+### **Want your open-source examples featured here?**
+Get in touch with us on the [#ComposeDB](https://discord.gg/6FCS44YDBs) Discord - we'd love to see what you're building!
```
Comment (JustinaPetr) [inline website/versioned_docs/version-0.5.x/examples.mdx:20]: Why do we call it a ComposeDB Discord? I think this should be Ceramic Discord instead. Unless you are referencing a composedb channel on discord (the invite link says that it's invalid so I can't really check).

I'd suggest referencing a general invite to our Discord https://chat.ceramic.network instead.

**Review by JustinaPetr — CHANGES_REQUESTED** [review]

File: website/versioned_docs/version-0.5.x/examples.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Starter Applications and Tutorials
+Looking for code samples, starter applications, and tutorials to kickstart your development process? Check out some of the examples below for inspiration. 
+
+## Starter Applications
+- [**Social App**](https://github.com/ceramicstudio/ComposeDbExampleApp) - A starter application built around a social media platform use-case (this application utilizes the same data models used by the [ComposeDB Sandbox](../../sandbox))
```
Comment (velvet-shark) [inline website/versioned_docs/version-0.5.x/examples.mdx:5]: Change "Social App" to "Official ComposeDB Example App"?

Comment (velvet-shark) [inline website/versioned_docs/version-0.5.x/examples.mdx:5]: "A starter application built around a social media platform use-case (this application utilizes the same data models used by the [ComposeDB Sandbox](../../sandbox)). A great first step if you haven't done anything with ComposeDB yet."

Comment (mzkrasner) [reply website/versioned_docs/version-0.5.x/examples.mdx:5]: Done!

Comment (mzkrasner) [reply website/versioned_docs/version-0.5.x/examples.mdx:5]: Thank you! Nice addition

**Review by velvet-shark — APPROVED** [review]

**Review by JustinaPetr — APPROVED** [review]

**Pull request merged**